examples/core/dof/app.js

```diff
@@ -225,7 +225,7 @@ void main() {
 `;
 
 export const animationLoopOptions = {
-  onInitialize: ({gl}) => {
+  onInitialize: ({gl, _animationLoop}) => {
```

Comment:
Unrelated nit: Perhaps time to drop the underscore on the `_animationLoop` field?

Reply:
I think so. I imagine it's not uncommon to want access to the AnimationLoop instance itself.